docs/content/en/docs/install/tips-tricks.md

```diff
@@ -0,0 +1,163 @@
+for an example.
+
+## Installing older versions of Keptn
+
+Installation of Keptn v0.9.0 has two significant differences
+compared to the installation of earlier releases:
+
+* Keptn v0.9.0 and later releases use
+  the umbrella charts whereas earlier versions did not
+* Keptn v0.9.0 and later releases use
+  `keptn` as the value to the `NAME` option of the
+  [helm repo add](https://helm.sh/docs/helm/helm_repo_add/)
+  command whereas earlier releases used `klt`.
+
+To install a version prior to v0.9.0,
+use the install command sequence that is documented for that release.
+Use the same command sequence documented for v0.9.0
+to determine the CHART version for the release you want to install:
+  
+```shell
+helm repo add keptn https://charts.lifecycle.keptn.sh
+helm repo update
+helm search repo keptn
+```
+
+To install an older release,
+specify that CHART version to the `--version` flag
+in the `helm update` command for the release you are installing.
+
+## Example configurations by use-case
+
+[Control what components are installed](install.md/#customizing-the-configuration-of-components)
+discusses how to configure Keptn to include only the components you want.
+The following sections summarize and give examples
+of the configurations needed for different use cases.
+
+### Enable Keptn Lifecycle Operator (Observability and/or Release Lifecycle)
+
+If you only want to run the Keptn Observability
+and/or Release Lifecycle use-cases in your cluster,
+you do not need to install the Keptn Metrics Operator.
+To disable it, set the `metricsOperator.enabled` value
+to `false` as in the following:
+
+{{< embed path="/docs/content/en/docs/install/assets/values-only-lifecycle.yaml" >}}
+
+Note that, if you want to run pre- and/or post-deployment
+[evaluations](../implementing/evaluations.md)
+as part of the Release Lifecycle use-case,
+you need to run the Keptn Metrics Operator.
+
+You must also enable Keptn for each
+[namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)
+on which you want to run either
+the Observability or Release Lifecycle use-case.
+This is because Keptn communicates with the Kubernetes scheduling mechanism
+for tasks such as enforcing checks natively,
+stopping a deployment from proceeding when criteria are not met,
+doing post-deployment evaluations
+and tracing all activities of all deployment
+[workloads](https://kubernetes.io/docs/concepts/workloads/)
+on the cluster.
+
+To enable Keptn in your namespace,
+annotate the appropriate `Namespace` resource(s).
+For example, for the `simplenode-dev` namespace,
+the annotation looks like this:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: simplenode-dev
+  annotations:
+    keptn.sh/lifecycle-toolkit: "enabled"
+```
+
+You see the annotation line `keptn.sh/lifecycle-toolkit: "enabled"`.
+This annotation tells the webhook to handle the namespace.
```

Comment:
it tells that the namespace is enabled for Keptn

Reply:
Anna added that.  I'm happy to delete.